enn_zoo/enn_zoo/vizdoom_env/vizdoom_config.ron

```diff
@@ -0,0 +1,28 @@
+ExperimentConfig(
+    total_timesteps: 500000,
+    env: (
+        id: "DoomBasic",
+    ),
+    rollout: (
+        num_envs: 16,
+        steps: 128,
+        processes: 1,
+    ),
+    net: (
+        d_model: 128,
+        n_layer: 4,
+        translation: (
+            reference_entity: "Player",
+            position_features: ["x", "y"],
+            rotation_angle_feature: "angle"
+        ),
+    ),
+    optim: (
+        bs: 1024,
+        lr: 0.00025,
+        update_epochs: 10
```

Comment:
that's a lot of epochs! 😮

Reply:
I used SB3 default PPO settings as a baseline, which seem to work well for ViZDoom when using images. Kinda seems to work here, but indeed makes training slower :D